File: python/util/factors.py
import math


def get_proper_divisors(num):
	divisors = [1]

	for i in range(2, math.floor(math.sqrt(num)) + 1):
		if num % i == 0:
			divisors.append(i)
			if num / i != i:
				divisors.append(num / i)

	return divisors

File: javascript/util/factors.js
function getProperDivisors(num) {
  const divisors = [1];

  for (let i = 2; i <= Math.sqrt(num); i++) {
    if (num % i === 0) {
      divisors.push(i);
      if (num / i !== i) {
        divisors.push(num / i);
      }
    }
  }

  return divisors;
}

module.exports = getProperDivisors;
